en/docs/fragments/enable-account-locking.md

```diff
@@ -0,0 +1,56 @@
+1. 	Ensure that the identity listener with the
+   `              priority=50             ` is set to **false** and
+   the identity listener with the `              priority=95             ` is set to
+   **true**  by adding the following configuration to the
+   `              <IS_HOME>/repository/conf/deployment.toml             ` file.  
+	``` xml
+	[event.default_listener.identity_mgt]
+	priority= "50"
+	enable = false
+	[event.default_listener.governance_identity_mgt]
+	priority= "95"
+	enable = true
+   	```
```

Comment:
need to say that if these configs are not changed in deployment.toml then no need to add as it as this is the default product behaviour